docs/modules/ROOT/pages/support/data-privacy.adoc

```diff
@@ -10,12 +10,6 @@ If you use a public GitHub repository as a component’s source, {ProductName} p
 
 *Note:* There is no functionality to use a private Quay repository. The artifacts {ProductName} builds are public and visible to anyone who is viewing the Quay repository. 
 
-== Data in GitHub repositories is public
```

Comment:
These are the GitHub GitOps repositories that HAS creates.  They're still public.

Reply:
thanks for your review, @amfred! 

what is "HAS"?

Reply:
Hyrbid Application Service ( HAS ) is the software component which sets creates these artifacts.